index.html

```diff
@@ -0,0 +1,405 @@
+<!DOCTYPE html>
```

Comment:
### Optional change 
- [ ] Validating the markup is a great way of ensuring it follows the best semantic convention. Feel free to use this [tool](https://validator.w3.org/nu/#textarea).